cmd/root.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "verify",
+	Short: "New Manifest Verification Tool Prototype",
+	Long:  `Some meaningful long description about the tool.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		// Do Stuff Here
```

Comment:
Wrong comment. Please either remove the comment if none is needed or be more descriptive.

Reply:
You forgot to address this comment

Reply:
Done.